src/components/Publish/Submission/GasFees.tsx

```diff
@@ -1,9 +1,28 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import styles from './GasFees.module.css'
 import Conversion from '@shared/Price/Conversion'
+import { useFormikContext } from 'formik'
+import { FormPublishData } from '../_types'
 
 export default function GasFees({ gasFees }: { gasFees: string }) {
-  return (
+  const { setErrors } = useFormikContext<FormPublishData>()
+  useEffect(() => {
+    if (gasFees === 'insufficient-funds') {
+      setErrors({
+        feedback: {
+          '1': {
+            status: 'error'
+          }
+        }
+      })
+    }
+  }, [gasFees])
+
+  return gasFees === 'insufficient-funds' ? (
+    <span className={`${styles.gasFees} ${styles.noFunds}`}>
+      Insufficient ETH balance
```

Comment:
This check seems to be exclusively for the gas fees, so we should say that, like "Insufficient ETH balance to cover gas fees".

Then, we only have ETH on the ETH mainnet and testnets, but not on all the other production/test networks for which we need to detect the native token of that chain and display it here

Reply:
no, on the pricing screen we just check if the user has enough ocean to add as liquidity, it does not handle the actual gas cost. I think we have the chain token when we get various details regarding the chain.

Reply:
ok, now I see it. The pricing screen seems to be checking the user's balance, but only for the "Dynamic" pricing tab. 

<img width="1174" alt="Screenshot 2022-02-21 at 15 02 19" src="https://user-images.githubusercontent.com/13741335/154970701-0cf359e6-5711-4fbe-a984-289dec9a12f3.png">
<img width="1172" alt="Screenshot 2022-02-21 at 15 02 25" src="https://user-images.githubusercontent.com/13741335/154970707-ebc0d3c7-20d8-4f8e-b8c2-adf3ee5c453f.png">


It seems there's a bug on it, because I can go to the next step anyway. I think is better to do this on the pricing screen as planned, I'll just remove my calculations and fix what's not working. is that ok?

Reply:
> on the pricing screen we just check if the user has enough ocean to add as liquidity, it does not handle the actual gas cost

yup, that's totally fine but yes, the continue button on there should be disabled if there's not enough baseToken in user's wallet.

Fyi, this only concerns the dynamic tab because with that pricing method user adds liquidity when creating the pool so the baseToken amount entered in that dynamic screen will e sent out of user's wallet (hence the approve tx before creating pool). This all doesn't apply to the other pricing methods

Reply:
> no, on the pricing screen we just check if the user has enough ocean to add as liquidity, it does not handle the actual gas cost. I think we have the chain token when we get various details regarding the chain.

got it. still, shouldn't this check on OCEAN balance be also done on the "Fixed" and "Free" tabs?